src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXNewServerStep/help_de.html

```diff
@@ -20,9 +20,6 @@
 ATX.newServer(String atxName, String toolName, ATXConfig config) : ATXServer
             </pre>
             <pre>
-ATX.newServer(String atxName, String toolName, String fullServerURL, boolean uploadToServer, String authKey, String projectId) : ATXServer
```

Comment:
this [should work anyway](https://github.com/jenkinsci/ecutest-plugin/blob/316b993ee7b7795e1527169531a855d972a7a590/src/main/java/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXPipeline.java#L131)